app/org/sagebionetworks/bridge/services/UploadService.java

```diff
@@ -41,19 +47,24 @@
     private UploadDao uploadDao;
     private Validator validator;
 
+    @Autowired
     public void setUploadSessionCredentialsService(UploadSessionCredentialsService uploadCredentialsService) {
         this.uploadCredentailsService = uploadCredentialsService;
     }
+    @Autowired
     public void setS3UploadClient(AmazonS3 s3UploadClient) {
         this.s3UploadClient = s3UploadClient;
     }
+    @Autowired
     public void setS3Client(AmazonS3 s3Client) {
         this.s3Client = s3Client;
     }
```

Comment:
We have multiple S3 clients (and in this case, 2 S3 clients in the same class). You'll probably want to use @Resource(name = "s3Client") and @Resource(name = "s3UploadClient") to disambiguate.